### ISSUE TITLE
Add a button to download a note as a .txt or a .md.

### ISSUE THREAD

**Issue by vega-d** [nextcloud/notes#766]
**Is your feature request related to a problem? Please describe.**
I'm a student and use Notes app to write my assignments, and to submit them I need to have them as a file. Right now I have to either go into Files tab, find my Note and download it, or select all the text, open it in a text editor on my computer, paste it there, save and then upload.  

**Describe the solution you'd like**
I would really prefer having a button I can just click/tap and it downloads a .txt or a .md file. 

**Additional context**
Here is a concept of how the button would look:
![image](https://user-images.githubusercontent.com/35414314/138602566-3fa30c22-0735-4052-8103-29991a909c21.png)

**Comment by korelstar** [comment]
If you are working often with files, it could be easier if you synchronize your Nextcloud files with your computer. There are synchronization tools for this purpose (e.g. Nextcloud desktop app or many other WebDAV clients).

I don't think that a "Download" button is needed by many people, therefore I don't like to add another menu item.

**Comment by vega-d** [comment]
> If you are working often with files, it could be easier if you synchronize your Nextcloud files with your computer.

Well, the thing is, I use Notes app in my nextcloud mainly because I can access it on any computer, not just mine, where I can install any software I want. 

If I was working with local files I'd just use notepad or something.

**Comment by SyndicatedPillbug** [comment]
Yeah, this would be a really nice feature. Not all of my users are in positions to download the main Nextcloud application are are using NextCloud _only_ from the graphical interface. The ability to download as a text file or PDF would be extremely helpful.
